lib/patch-https-request-for-proxying.js

```diff
@@ -30,7 +30,7 @@ function parseNoProxyZone(zone) {
   return {hostname: zoneHost, port: zonePort, hasPort: hasPort};
 }
 
-function uriInNoProxy(uri, noProxy) {
+function uriInNoProxy(uri) {
   const port = uri.port || (uri.protocol === 'https:' ? '443' : '80');
   const hostname = formatHostname(uri.hostname);
   const noProxyList = noProxy.split(',');
```

Comment:
where this `noProxy` is coming from now?

Reply:
It's imported at the top of the file, and the call site for this function was just passing it through. Now this function just uses the globally available one rather than having it passed in.